### PR TITLE
test

### PR DESCRIPTION
**Pull request by vinokurig**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
 # hello-world-backend-plugin
 hello-world-backend-plugin Plugin example for Theia.
 
-commit111111111
\ No newline at end of file
+test
```
Comment (vinokurig) [inline README.md:4]: Test comment

Comment (vinokurig) [reply README.md:4]: New comment

Comment (vinokurig) [reply README.md:4]: replay

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
 # hello-world-backend-plugin
 hello-world-backend-plugin Plugin example for Theia.
```
Comment (vinokurig) [inline README.md:2]: ffff

Comment (vinokurig) [inline README.md:2]: ff

Comment (vinokurig) [inline README.md:2]: ff

Comment (vinokurig) [inline README.md:2]: ffff

Comment (vinokurig) [inline README.md:2]: qqqqq

Comment (vinokurig) [inline README.md:2]: qqqqq

Comment (vinokurig) [inline README.md:2]: qqqqq

File: README.md (hunk 1 above)
Comment (vinokurig) [reply README.md:4]: ff

File: README.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
...
```
Comment (vinokurig) [inline README.md:3]: Hello

Comment (vinokurig) [inline README.md:3]: qqqqqqq

File: README.md (hunk 2 above)
Comment (vinokurig) [inline README.md:2]: qqqqqqqq

Comment (vinokurig) [inline README.md:2]: qqqqqq

Comment (vinokurig) [reply README.md:2]: qqqqq

Comment (vinokurig) [inline README.md:2]: aaaaaa

Comment (vinokurig) [reply README.md:2]: qqqq

Comment (vinokurig) [inline README.md:2]: qqqqqq

Comment (vinokurig) [reply README.md:2]: q

Comment (vinokurig) [inline README.md:2]: qqqq

Comment (vinokurig) [reply README.md:2]: qq

Comment (vinokurig) [reply README.md:2]: qqqqqq

Comment (vinokurig) [reply README.md:2]: q

Comment (vinokurig) [inline README.md:2]: qqqqq

Comment (vinokurig) [inline README.md:2]: qqqqq

Comment (vinokurig) [inline README.md:2]: Hello

Comment (vinokurig) [inline README.md:2]: qqqqqq

Comment (vinokurig) [reply README.md:2]: qq

Comment (vinokurig) [inline README.md:2]: qwe

Comment (vinokurig) [reply README.md:2]: ewq

Comment (vinokurig) [inline README.md:2]: qwe

Comment (vinokurig) [reply README.md:2]: ewq

Comment (vinokurig) [inline README.md:2]: qwe

Comment (vinokurig) [reply README.md:2]: wqe

Comment (vinokurig) [inline README.md:2]: qwe

Comment (vinokurig) [reply README.md:2]: ffff
fffff
ff

Comment (vinokurig) [reply README.md:2]: qqq
qq
q

Comment (vinokurig) [reply README.md:2]: qqqqq

Comment (vinokurig) [reply README.md:2]: 123456

Comment (vinokurig) [reply README.md:2]: qqqqq

File: README.md (hunk 1 above)
Comment (vinokurig) [inline README.md:4]: asddddddddd

File: README.md (hunk 2 above)
Comment (vinokurig) [reply README.md:2]: qwqer

File: README.md (hunk 1 above)
Comment (vinokurig) [inline README.md:4]: aaa

File: README.md (hunk 2 above)
Comment (vinokurig) [inline README.md:2]: New comment

Comment (vinokurig) [inline README.md:2]: ffff

File: README.md (hunk 1 above)
Comment (vinokurig) [inline README.md:4]: ddddd

File: README.md (hunk 2 above)
Comment (vinokurig) [inline README.md:2]: New comment

Comment (vinokurig) [reply README.md:2]: aaaa

Comment (vinokurig) [reply README.md:2]: qqqqq

Comment (vinokurig) [reply README.md:2]: ddddd

Comment (vinokurig) [reply README.md:2]: qqqqqq

Comment (vinokurig) [reply README.md:2]: qq

Comment (azatsarynnyy) [reply README.md:2]: checking...

Comment (vinokurig) [reply README.md:2]: test

Comment (testmagic-123) [reply README.md:2]: fffff

Comment (testmagic-123) [reply README.md:2]: new

Comment (testmagic-123) [reply README.md:2]: test

File: README.md (hunk 1 above)
Comment (testmagic-123) [inline README.md:4]: Hello

Comment (testmagic-123) [reply README.md:4]: new

File: README.md (hunk 2 above)
Comment (testmagic-123) [reply README.md:2]: test

Comment (testmagic-123) [inline README.md:2]: review

Comment (testmagic-123) [reply README.md:2]: end

Comment (testmagic-123) [reply README.md:2]: test

Comment (testmagic-123) [reply README.md:2]: test

Comment (testmagic-123) [reply README.md:2]: test

Comment (testmagic-123) [reply README.md:2]: new

Comment (testmagic-123) [reply README.md:2]: new

Comment (testmagic-123) [reply README.md:2]: new

Comment (testmagic-123) [reply README.md:2]: dddd

Comment (testmagic-123) [reply README.md:2]: dddd

Comment (testmagic-123) [reply README.md:2]: qqq

Comment (testmagic-123) [reply README.md:2]: qqqq

Comment (testmagic-123) [reply README.md:2]: ffff

Comment (testmagic-123) [reply README.md:2]: qqq

Comment (vinokurig) [reply README.md:2]: qqqq

Comment (vinokurig) [reply README.md:2]: qqqq

Comment (vinokurig) [reply README.md:2]: ssss

Comment (vinokurig) [reply README.md:2]: aaa

Comment (vinokurig) [reply README.md:2]: qqq

Comment (vinokurig) [reply README.md:2]: qqqq

Comment (testmagic-123) [reply README.md:2]: new

Comment (testmagic-123) [reply README.md:2]: new

Comment (testmagic-123) [reply README.md:2]: new

Comment (testmagic-123) [reply README.md:2]: new

Comment (testmagic-123) [reply README.md:2]: new

Comment (testmagic-123) [reply README.md:2]: aaaaa

Comment (testmagic-123) [reply README.md:2]: qqqq

Comment (testmagic-123) [reply README.md:2]: Another one

Comment (testmagic-123) [reply README.md:2]: new

Comment (testmagic-123) [reply README.md:2]: new

Comment (testmagic-123) [reply README.md:2]: new

Comment (testmagic-123) [reply README.md:2]: new

Comment (testmagic-123) [reply README.md:2]: new

Comment (testmagic-123) [reply README.md:2]: new

Comment (testmagic-123) [reply README.md:2]: new

Comment (testmagic-123) [reply README.md:2]: new

Comment (testmagic-123) [reply README.md:2]: new

Comment (RomanNikitenko) [reply README.md:2]: test comment

Comment (RomanNikitenko) [reply README.md:2]: new test comment

Comment (RomanNikitenko) [reply README.md:2]: test

Comment (RomanNikitenko) [reply README.md:2]: edit comment

Comment (testmagic-123) [reply README.md:2]: Third comment

Comment (azatsarynnyy) [reply README.md:2]: single comment testing

Comment (azatsarynnyy) [inline README.md:2]: review comment 1 (edited)

File: README.md [hunk 4]
```diff
@@ -1,4 +1,4 @@
 # hello-world-backend-plugin
```
Comment (azatsarynnyy) [inline README.md:1]: review comment 2

File: README.md (hunk 1 above)
Comment (azatsarynnyy) [inline README.md:4]: comment 3

File: README.md [hunk 5]
```diff
@@ -1,2 +1,4 @@
...
```
Comment (vinokurig) [inline README.md:1]: dddddd

File: README.md [hunk 6]
```diff
@@ -1,2 +1,4 @@
...
 hello-world-backend-plugin Plugin example for Theia.
+
+test
```
Comment (vinokurig) [inline README.md:4]: ddddd

Comment (vinokurig) [inline README.md:4]: ddddd

File: README.md (hunk 5 above)
Comment (vinokurig) [inline README.md:1]: 12

File: README.md (hunk 6 above)
Comment (vinokurig) [inline README.md:4]: ffff

Comment (vinokurig) [reply README.md:4]: ff

Comment (vinokurig) [inline README.md:4]: New comment

File: README.md (hunk 2 above)
Comment (balajiv113) [reply README.md:2]: Forth Comment

**Pull request closed without merging**